src/com/rcv/ElectionConfig.java

```diff
@@ -82,4 +88,36 @@ public boolean treatBlankAsUWI() {
       false :
       rawConfig.rules.treat_blank_as_uwi;
   }
+
+  // returns list of strings for use in matching CVR cells to candidates
+  // looks for candidate code first and fallback to candidate name to support both variants
+  public List<String> getCandidateCodeList() {
+    return mCandidateCodeList;
+  }
+
+  // returns String for display in the visualizer
+  public String getNameForCandidateID(String candidateID) {
+    return mCandidateCodeToNameMap.get(candidateID);
+  }
+  
+  private ArrayList<String> mCandidateCodeList = null;
```

Comment:
Any reason not to declare these at the top of the class?

Reply:
ok